prow/pluginhelp/pluginhelp.go

```diff
@@ -41,6 +57,8 @@ type PluginHelp struct {
 	// Events is a slice containing the events that are handled by the plugin.
 	// NOTE: Plugins do not need to populate this. Hook populates it on their behalf.
 	Events []string
+	// Commands maps a command name to a struct of its properties.
+	Commands map[string]Command
```

Comment:
This might be more appropriate as a slice of `Command`s. We don't have names assigned to each command, the usage strings should be sufficient.

Reply:
What do you mean by "the usage strings should be sufficient"? My intent is to replace the usage string by the `Commands` map. We dont have name for each command now, but we can config using the method `AddCommand()` below. This makes changes to every single plugin  file and will be done after this is checked in.

Reply:
I just meant that we don't have names for each command right now and that I don't see a reason to name them. The usage string seems like an easy enough way to identify the command. The `AddCommand` method would just take a `Command` instead of a name and a `Command`.